filament/src/MorphTargetBuffer.cpp

```diff
@@ -74,114 +77,102 @@ static inline size_t getHeight(size_t vertexCount) noexcept {
     return (vertexCount + MAX_MORPH_TARGET_BUFFER_WIDTH) / MAX_MORPH_TARGET_BUFFER_WIDTH;
 }
 
-static inline size_t getDepth(size_t targetCount) noexcept {
-    return targetCount * 2;
-}
+template<VertexAttribute A>
+inline size_t getSize(size_t vertexCount) noexcept;
 
-static inline size_t getSize(size_t vertexCount) noexcept {
-    const size_t stride = getWidth(vertexCount) * sizeof(float4);
+template<>
+inline size_t getSize<VertexAttribute::POSITION>(size_t vertexCount) noexcept {
+    const size_t stride = getWidth(vertexCount) * sizeof(float3);
+    const size_t height = getHeight(vertexCount);
     return Texture::PixelBufferDescriptor::computeDataSize(
-            Texture::PixelBufferDescriptor::PixelDataFormat::RGBA,
+            Texture::PixelBufferDescriptor::PixelDataFormat::RGB,
```

Comment:
Are we sure we want RGB here? I'm not sure RGB formats are natively supported by any driver/GPU,, so it might turn into RGBA anyway. Besides I think the Vulkan/Metal backends might just expand to RGBA, and it would be more expensive to do it in two steps. @prideout and @bejado to check me

Reply:
Yes you are right. I will revert it. Thanks.